sirepo/srschema.py

```diff
@@ -164,6 +165,15 @@ def validate(schema):
         for src in schema.dynamicModules[t]:
             pkresource.filename(src[1:])
     _validate_strings(schema.strings)
+    _validate_constants_methods(schema.constants.get('methods', PKDict()))
+
+
+def _validate_constants_methods(methods):
```

Comment:
Any validation should be against the template itself: do these methods exist? Which then begs the question, why not just generate the list from the template, which then begs the question: why bother?

Reply:
d6fccd452